src/components/Api.js

```diff
@@ -0,0 +1,181 @@
+export default class Api {
+  constructor({ baseURL, authToken }) {
+    this._baseURL = baseURL;
+    this._authToken = authToken;
+  }
+
+  // GET https://around.nomoreparties.co/
+  getCardList() {
+    return fetch("https://around.nomoreparties.co/v1/group-12/cards", {
+      headers: {
+        authorization: this._authToken,
+      },
+    }).then((res) => {
```

Comment:
The code for processing server response gets duplicated. You should declare a method and reuse it

```js
_processResponse = (res) => { 
      if (res.ok) { 
        return res.json(); 
      } 
      return Promise.reject(`Error: ${res.status}`); 
}

getInitialCards() { 
  ...
  .then(this._processResponse)
}
```